src/parse.c

```diff
@@ -2926,6 +2926,9 @@ handle_network_ovs_settings_global_ports(NetplanParser* npp, yaml_node_t* node,
         peer = yaml_document_get_node(&npp->doc, *(item+1));
         assert_type(npp, peer, YAML_SCALAR_NODE);
 
+        if (!g_strcmp0(scalar(port), scalar(peer)))
+            return yaml_error(npp, peer, error, "Openvswitch peer ports must be different");
```

Comment:
nitpick: improve error message a bit.
```suggestion
            return yaml_error(npp, peer, error, "Open vSwitch patch ports must be of different name");
```

We use the "patch port" term in our documentation, so should be using it here, too (and probably in the other OVS error messages). And we should also use the official "Open vSwitch" writing for OVS (same with the other error messages).